specification/definitions/Service.json

```diff
@@ -55,11 +55,11 @@
           "additionalProperties": false,
           "properties": {
             "min": {
-              "type": ["number", "null"],
+              "type": "number",
```

Comment:
Not a comment for this line, but `transit_time` is currently a required property, this should not be required (we don't send`null` values).